packages/slice-machine/src/features/customTypes/customTypesBuilder/GenerateSliceWithAiModal/GenerateSliceWithAiModal.tsx

```diff
@@ -125,11 +131,11 @@ export function GenerateSliceWithAiModal(props: GenerateSliceWithAiModalProps) {
         <DialogActions>
           <DialogCancelButton disabled={isCreatingSlices} />
           <DialogActionButton
-            disabled={!allSlicesReady}
+            disabled={!someSlicesReady}
```

Comment:
why not wait for them to finish before enabling the button? if we do this, then i think we need to cancel ongoing uploads on submit to make sure that things don't continue working in the background

Reply:
Good catch. I fixed it to disable when any slice is loading ☑️